index.php

```diff
@@ -38,17 +39,9 @@
 						<h2>View Portfolio</h2>
 					</a>
 				</div>
-				<h4 id="about">My name is Kyam Harris, a father of two from
-					Somerset currently training to become a Full Stack Web
-					Developer with Mayden Academy.
-					I’ve spent my whole career in Customer Service, but recently
-					discovered that I enjoyed using computers to solve problems
-					at work and, after tinkering with Python, that I enjoy programming!
-				</h4>
-				<p>
-					I have been accepted to join Mayden Academy for 16 weeks to
-					learn HTML, CSS, Javascript and PHP, but more importantly to
-					think and code like a true programmer.
+				    <h4 id="about"><?php echo $abouts[2]['content']?>
```

Comment:
Ok. So it looks like you don't always need the first item of the array but I left the comment above in case it's useful for another time